values/drone/drone-admit-members.gotmpl

```diff
@@ -29,4 +29,5 @@ podAnnotations:
 image:  
   pullSecrets:
     - name: otomi-pullsecret-global
+    - name: gcr-json-key
```

Comment:
Why are you adding a pullsecret here for otomi api?

Reply:
I had to add this Secret in `otomi-eks-dev` or else it threw a pull error, it doesn't make sense to you? I am curious what the alternative solution is (also for my learning experience)?

Reply:
I didn't understand @Morriz comment, but he means like "why is it added in this namespace", looking at it now I wonder if it is relevant in this context? Did you verify it @mojtabaimani?